pip/compat/dictconfig.py

```diff
@@ -153,8 +153,8 @@ class BaseConfigurator(object):
     DIGIT_PATTERN = re.compile(r'^\d+$')
 
     value_converters = {
-        'ext' : 'ext_convert',
-        'cfg' : 'cfg_convert',
+        'ext': 'ext_convert',
+        'cfg': 'cfg_convert',
     }
```

Comment:
This is a copy of the Python logging.config.dictconfig module as stated at the beginning of the file.
So we won't modify anything.